src/quick-lint-js/fe/parse-statement.cpp

```diff
@@ -4851,6 +4845,13 @@ void Parser::parse_and_visit_import(
           //
           // FIXME(strager): Should this behave like an import or an import
           // alias or some other kind?
```

Comment:
Nit: This comment should be attached to line 4855. I'd move your new `if` statement (lines 4848-4854) above this comment (line 4842).

Reply:
Done!